framework/src/modules/random/module.ts

```diff
@@ -25,15 +25,16 @@ import {
 } from '../../state_machine';
 import { BaseModule, ModuleInitArgs, ModuleMetadata } from '../base_module';
 import { RandomAPI } from './api';
-import { defaultConfig, EMPTY_KEY, STORE_PREFIX_USED_HASH_ONION } from './constants';
+import { defaultConfig, EMPTY_KEY } from './constants';
 import { RandomEndpoint } from './endpoint';
 import {
 	blockHeaderAssetRandomModule,
 	isSeedRevealValidRequestSchema,
 	isSeedRevealValidResponseSchema,
 	randomModuleConfig,
 } from './schemas';
-import { BlockHeaderAssetRandomModule, UsedHashOnion } from './types';
+import { BlockHeaderAssetRandomModule } from './types';
+import { UsedHashOnion } from './stores/used_hash_onions';
```

Comment:
I think you can combine this import and the one on line 41 together as they import from the same file. e.g. 
`import { UsedHashOnion , UsedHashOnionsStore } from './stores/used_hash_onions';`